### PR TITLE
adding AAC page

### PR DESCRIPTION
**Pull request by nycbeardo**
added page for American Alpine Club and associated img and posts

# Description

adding statements from American Alpine Club from social media pages to its own markdown file

Fixes # (issue)

## Data added

Please delete options that are not relevant.

- [x] Company data addition

# How Has This Been Tested?

Tested in remote config in Google Chrome

**Test Configuration**:
* Browser: Tested in Chrome

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: companies/american-alpine-club.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    date_posted: 2020-06-02
+    type: Blog
+    raw_text: >
+     Black lives matter—throughout the country and at the crag. The American Alpine Club stands in solidarity with those protesting systemic racism on the frontlines and from home. As climbers and as humans, we believe that racism has no place in our craft or our country.
+     
+     Please read our full statement, "The AAC Stands in Solidarity."
+     
+     https://bit.ly/2BmOZFf
+exec_team_url: https://americanalpineclub.org/staff
+diversity_inclusion_url: https://americanalpineclub.org/news/2020/6/2/the-aac-stands-in-solidarity?fbclid=IwAR0eNytmunD0H0s915uGahQA069P4UxdTeMU14a7Q_uo0ELZ0kMTOxjE-Wc
```
Comment (kelseyhuse30) [inline companies/american-alpine-club.md:20]:
```suggestion
diversity_inclusion_url: https://americanalpineclub.org/news/2020/6/2/the-aac-stands-in-solidarity
```

It looks like we can use this URL without the extra query params. I need to add this step into the guide.

**Review by kelseyhuse30 — APPROVED** [review]
Nice work!  Thanks for finding that extra tweet 👍🏻 👍🏻 

This looks really good, there are a few lint errors from yamllint.  I'll leave it up to you if you wanna fix those but it would help us stay consistent.

Thanks for contributing to our project! 👏🏻 💓

File: companies/american-alpine-club.md (hunk 1 above)
Comment (nycbeardo) [reply companies/american-alpine-club.md:20]: Url changed to reflect deletion of query params, please see if I did it correctly

Comment (kelseyhuse30) [reply companies/american-alpine-club.md]: yep that's perfect

Comment (nycbeardo) [reply companies/american-alpine-club.md]: yamllint errors fixed, can it be merged with main?

Comment (nycbeardo) [reply companies/american-alpine-club.md]: I'm still getting this one pesky error about the wrong syntax on line 10, can you check to see if I overlooked something?

File: companies/american-alpine-club.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: American Alpine Club
+image: american-alpine-club.png
+blm_statements:
+  - url: https://twitter.com/americanalpine/status/1267946389482266625
+    date_posted: 2020-06-02
+    type: Tweet
+    raw_text: >
+    Black lives matter—throughout the country and at the crag. The AAC stands in solidarity with those protesting systemic racism on the frontlines and from home.
```
Comment (kelseyhuse30) [inline companies/american-alpine-club.md:9]:
```suggestion
      Black lives matter—throughout the country and at the crag. The AAC stands in solidarity with those protesting systemic racism on the frontlines and from home.
```

File: companies/american-alpine-club.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - url: https://www.facebook.com/AmericanAlpineClub/photos/a.310680388934/10157980875738935
+    date_posted: 2020-06-02
+    type: Blog
+    raw_text: >
+    Black lives matter—throughout the country and at the crag. The American Alpine Club stands in solidarity with those protesting systemic racism on the frontlines and from home. As climbers and as humans, we believe that racism has no place in our craft or our country.
```
Comment (kelseyhuse30) [inline companies/american-alpine-club.md:14]:
```suggestion
      Black lives matter—throughout the country and at the crag. The American Alpine Club stands in solidarity with those protesting systemic racism on the frontlines and from home. As climbers and as humans, we believe that racism has no place in our craft or our country.
```

File: companies/american-alpine-club.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    Please read our full statement, "The AAC Stands in Solidarity." https://bit.ly/2BmOZFf
```
Comment (kelseyhuse30) [inline companies/american-alpine-club.md:15]:
```suggestion
      Please read our full statement, "The AAC Stands in Solidarity." 
      https://bit.ly/2BmOZFf
```

**Comment by kelseyhuse30** [comment]
@nycbeardo  I _think_ the raw_text needs indented by 2 more spaces.  Hopefully that'll fix it 🤞🏻 

This style of formatting is maybe not the best lol I am now considering looking into switching to JSON instead.

File: companies/american-alpine-club.md (hunk 2 above)
Comment (nycbeardo) [reply companies/american-alpine-club.md]: I was able to fix the errors, acceptable/safe to merge or no?

**Comment by kelseyhuse30** [comment]
@nycbeardo thanks for fixing those lint errors! Glad you contributed to the project and we appreciate your work 👏🏻 💓 

Your page is now live here -> https://diversitycorp.co/companies/american-alpine-club/

**Pull request merged**